Common/src/main/java/at/petrak/hexcasting/common/casting/actions/spells/OpFlight.kt

```diff
@@ -35,11 +35,20 @@ class OpFlight(val type: Type) : SpellAction {
         val theArg = args.getPositiveDouble(1, argc)
         env.assertEntityInRange(target)
 
-        val cost = when (this.type) {
+        /*val cost = when (this.type) {
             Type.LimitRange -> theArg * MediaConstants.DUST_UNIT
             // A second of flight should cost 1 shard
             Type.LimitTime -> theArg * MediaConstants.SHARD_UNIT
-        }.roundToLong()
+        }.roundToLong()*/
+        val cost = if(target.abilities.mayfly){
+            0L
```

Comment:
could you add a comment here either explaining why it's 0 or just linking to this PR?

Reply:
👍